backend/app/routes/auth.py

```
@@ -0,0 +1,75 @@
+from fastapi import APIRouter, HTTPException, Depends
```

Ruff (0.8.2)

`fastapi.Depends` imported but unused

Remove unused import: `fastapi.Depends`

(F401)